src/libs/GrowlNotification/index.js

```diff
@@ -0,0 +1,89 @@
+import React, {Component} from 'react';
+import {
+    Text, View, Animated,
+} from 'react-native';
+import {
+    Directions, FlingGestureHandler, State, TouchableWithoutFeedback,
+} from 'react-native-gesture-handler';
+import colors from '../../styles/colors';
+import Icon from '../../components/Icon';
+import {Checkmark, Exclamation} from '../../components/Icon/Expensicons';
+import styles from '../../styles/styles';
+import GrowlNotificationContainer from './GrowlNotificationContainer';
+
+const types = {
+    success: {
+        icon: Checkmark,
+        iconColor: colors.green,
+    },
+    error: {
+        icon: Exclamation,
+        iconColor: colors.red,
+    },
+    warning: {
+        icon: Exclamation,
+        iconColor: colors.yellow,
+    },
+};
+
+const outDistance = -255;
+
+class GrowlNotification extends Component {
+    constructor() {
+        super();
+
+        this.state = {
+            bodyText: '',
+            type: 'success',
+            translateY: new Animated.Value(outDistance),
+        };
+    }
+
+    show = (bodyText, type, duration = 2000) => {
```

Comment:
For class components, you should use regular (non-arrow) member functions and then bind them to `this` in the constructor:

```suggestion
    show(bodyText, type, duration = 2000) {
```

Also, please add JSDoc comments to all the member functions 😄

Reply:
Done